CMSgov/onemac
# Oy2 20112 and Oy2 16977

Story: https://qmacbis.atlassian.net/browse/OY2-20112
- also covers https://qmacbis.atlassian.net/browse/OY2-16977
Endpoint: See github-actions bot comment

### Details

Create Submission Form: RAI Response for 1915(c) Appendix K

### Changes

- Added form for Appendix K RAI Response
- Added action to kebab and 'Actions' section in package details to get to the form

### Implementation Notes

- Form was modeled after other RAI response forms
- OY2-16977 was rolled into this story since the Links to the form were necessary to get to the form easily
- Added to serverless.yml for the AppKRAI submit

### Test Plan

1. Login 
2. Find an App K in RAI issued status
3. Click the kebab or navigate to the 'Package Details' and click on 'Respond to RAI'
4. Attach necessary attachment, and additional info if necessary 
5. Click submit
6. Should see green success popup and if you are logged in a real ID (not statesubmitter) you should receive an email
